model/post.go

```diff
@@ -390,3 +401,76 @@ func (o *Post) GenerateActionIds() {
 		}
 	}
 }
+
+var markdownDestinationEscaper = strings.NewReplacer(
+	`\`, `\\`,
+	`<`, `\<`,
+	`>`, `\>`,
+	`(`, `\(`,
+	`)`, `\)`,
+)
+
+func (o *Post) WithRewrittenImageURLs(f func(string) string) *Post {
```

Comment:
`f` Is a bad name for a parameter.

Reply:
If you think it's unclear, I'm open to renaming or documenting things for clarification, but `f` is at least idiomatic. Just did a search through the standard library and the vast majority of function arguments are simply named `f` (e.g. see the `strings` package). I think the idea is that argument's purpose should be clear from the function name, and if it's not, the function name (not just the argument name) is probably bad.

Reply:
`func (o *Post) WithRewrittenImageURLs(f func(string) string) *Post`  I don't think it's clear what `f` is for just looking at the signature. I would rename or add a doc comment.

I also hate 1 letter variables anywhere and try to avoid them.

Reply:
Sure. I'll at least add documentation and try to think of other names. 👍